.gitignore

```diff
@@ -13,7 +13,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-/sessions
+/sessions/*.json

```

Comment:
This is to have the sessions folder in the repo so that there are no EPERM issue when creating session files